### PR TITLE
Split deployment release notes by target platform

### PR DESCRIPTION
**Pull request by MarkvanMents**

### CODE REVIEW

File: content/releasenotes/developer-portal/deployment.md [hunk 1]
```diff
@@ -2,11 +2,29 @@
...
+| [Other Deployment Options](on-premises) | October 26th, 2020 |
+
+The release notes for the buildpacks are held in their respective GitHub repos:
+
+* [Mendix Cloud Foundry Buildpack – cf-mendix-buildpack](https://github.com/mendix/cf-mendix-buildpack/releases)
+* [Mendix Buildpack for Docker – docker-mendix-buildpack](https://github.com/mendix/docker-mendix-buildpack/releases)
+
+To see the current status of the Mendix Clouds and Mendix for Private Cloud, see [Mendix Status](https://status.mendix.com/). Here you can also see planned maintenance and past incidents.
+
+<!-- This is what was in the release notes when they were split into separate categories
```
Comment (Adam-Dupaski) [inline content/releasenotes/developer-portal/deployment.md:27]: Are you going to remove this and move it somewhere as a backup?

File: content/releasenotes/developer-portal/mendix-for-private-cloud.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
+---
+title: "Mendix for Private Cloud"
+parent: "deployment"
+menu_order: 20
+description: "Release notes for deployment using Mendix for Private Cloud"
+tags: ["release notes", "deployment", "Mendix for Private Cloud", "Private Cloud"]
+---
+
```
Comment (Adam-Dupaski) [inline content/releasenotes/developer-portal/mendix-for-private-cloud.md:8]: A small intro as done for Mendix Cloud RN and/or a link to Private Cloud docs would be useful at the top here.

File: content/releasenotes/developer-portal/on-premises.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
+title: "Other Deployment Options"
+parent: "deployment"
+menu_order: 40
+description: "Release notes for On-premises deployments based on Virtual Machine (Windows and Linux) and Buildpacks (Cloud Foundry and Docker)"
+tags: ["release notes", "deployment", "on-premises", "buildpack", "cloud foundry", "docker", "IBM Cloud Portal"]
+---
+
```
Comment (Adam-Dupaski) [inline content/releasenotes/developer-portal/on-premises.md:8]: A small intro as done for Mendix Cloud RN and/or a link to respect docs would be useful at the top here.

File: content/releasenotes/developer-portal/sap-cloud-platform.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
+---
+title: "SAP Cloud Platform"
+parent: "deployment"
+menu_order: 30
+description: "Release notes for deployment to SAP Cloud Platform"
+tags: ["release notes", "deployment", "cloud environment", "SAP", "SAP Cloud"]
+---
+
```
Comment (Adam-Dupaski) [inline content/releasenotes/developer-portal/sap-cloud-platform.md:8]: A small intro as done for Mendix Cloud RN and/or a link to SAP docs would be useful at the top here.

File: content/releasenotes/developer-portal/deployment.md [hunk 5]
```diff
@@ -2,11 +2,29 @@
 title: "Deployment"
 category: "Developer Portal"
 description: "Release notes for deployment capabilities managed in the Mendix Developer Portal"
-tags: ["deployment", "cloud environment", "Mendix Cloud", "SAP", "SAP Cloud", "IBM", "on-premises", "free app"]
-#This document is mapped to the landing page, update the link there if renaming or moving the doc file.
+tags: ["release notes", "deployment", "cloud environment", "Mendix Cloud", "SAP", "SAP Cloud", "IBM", "on-premises", "free app"]
+#This document is mapped to the landing page, featured.html. Update the link there if renaming or moving the doc file.
```
Comment (Adam-Dupaski) [inline content/releasenotes/developer-portal/deployment.md:6]: Do you want to change the Latest Releases for deployment options (for example, at least add Mendix Cloud), or just leave it with a link to this doc and update the date there for every one of these releases?

File: content/releasenotes/developer-portal/deployment.md (hunk 1 above)
Comment (MarkvanMents) [reply content/releasenotes/developer-portal/deployment.md:27]: Yes, I'm just leaving it here so it is easy to find things and move them if the Product Owners have different ideas.
I'll remove the commented stuff before I publish.
Perhaps I'll add a {{To do}} section to remind me :-)

File: content/releasenotes/developer-portal/mendix-for-private-cloud.md (hunk 2 above)
Comment (MarkvanMents) [reply content/releasenotes/developer-portal/mendix-for-private-cloud.md:8]: Good idea - I'll add one.

File: content/releasenotes/developer-portal/deployment.md (hunk 5 above)
Comment (MarkvanMents) [reply content/releasenotes/developer-portal/deployment.md:6]: My idea was to leave just one link to deployment on the featured page pointing here and then make it more nuanced on this page so people know if they need to go further.
Perhaps I could use Google analytics to find out how many people navigate from featured to here. If there are a lot, I could reconsider.
@Adam-Dupaski - Do you think Connor would enjoy that, or shall I do it myself?

File: content/releasenotes/developer-portal/deployment.md (hunk 1 above)
Comment (MarkvanMents) [reply content/releasenotes/developer-portal/deployment.md:27]: Add a To Do to remind me.

File: content/releasenotes/developer-portal/mendix-for-private-cloud.md (hunk 2 above)
Comment (MarkvanMents) [reply content/releasenotes/developer-portal/mendix-for-private-cloud.md:8]: Added introduction

File: content/releasenotes/developer-portal/on-premises.md (hunk 3 above)
Comment (MarkvanMents) [reply content/releasenotes/developer-portal/on-premises.md:8]: Added introduction

File: content/releasenotes/developer-portal/sap-cloud-platform.md (hunk 4 above)
Comment (MarkvanMents) [reply content/releasenotes/developer-portal/sap-cloud-platform.md:8]: Added introduction

File: content/releasenotes/developer-portal/on-premises.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
+---
+title: "Other Deployment Options"
```
Comment (MarkvanMents) [inline content/releasenotes/developer-portal/on-premises.md:2]: Is there a better description for this - seems a bit of a catch-all

File: content/releasenotes/developer-portal/on-premises.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+parent: "deployment"
+menu_order: 40
+description: "Release notes for On-premises deployments based on Virtual Machine (Windows and Linux) and Buildpacks (Cloud Foundry and Docker)"
+tags: ["release notes", "deployment", "on-premises", "buildpack", "cloud foundry", "docker", "IBM Cloud Portal"]
+---
+
+These release notes cover changes to On-premises deployments based on Virtual Machine (Windows and Linux), deployment to [IBM Cloud Portal](/developerportal/deploy/ibm-cloud), and important customer-facing changes to the buildpacks. For full release notes for the buildpacks see the [Cloud Foundry](https://github.com/mendix/cf-mendix-buildpack/releases) and [Docker](https://github.com/mendix/docker-mendix-buildpack/releases) buildpack release notes on GitHub. There are separate release notes for other deployment targets, see [Deployment](deployment) release notes page for further information.
```
Comment (MarkvanMents) [inline content/releasenotes/developer-portal/on-premises.md:9]: There are only three notes here. Are they necessary?

File: content/developerportal/deploy/ibm-cloud.md [hunk 8]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "IBM Cloud"
 category: "Deployment"
-menu_order: 30
+menu_order: 65
```
Comment (MarkvanMents) [inline content/developerportal/deploy/ibm-cloud.md:4]: I've moved IBM Cloud down the list a bit as it doesn't seem to be being maintained in the same way as SAP and Private Cloud.

**Pull request merged**